.travis.yml

```diff
@@ -32,7 +32,7 @@ install:
   - npm install
 
 script:
-  - npm test
+  - $TRAVIS_BUILD_DIR/sysconfcpus/bin/sysconfcpus -n 2 npm test
```

Comment:
using `-n 1` is actually more performant than -n 2 :D

Reply:
oh, yes. forgot about that